Major Key/MajorKeyView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MajorKeyView.swift
+//  Major Key
+//
+//  Created by Patrick Steiner on 29.11.17.
+//  Copyright © 2017 Felix Krause. All rights reserved.
+//
+
+import UIKit
+
+protocol MajorKeyViewDelegate: class {
+    func didPressKeyButton(button: UIButton)
+    func didPressPackageButton(button: UIButton)
+}
+
+class MajorKeyView: UIView {
```

Comment:
What is this? Proper iOS architecture and stuff? hehe

Reply:
What do you mean? This is the new custom class which handles the 2 buttons in the inputAccessoryView.